opentelemetry-api/src/opentelemetry/context/__init__.py

```diff
@@ -172,3 +172,7 @@ def detach(token: object) -> None:
 _SUPPRESS_HTTP_INSTRUMENTATION_KEY = create_key(
     "suppress_http_instrumentation"
 )
+
+__all__ = [
+    "Context",
```

Comment:
I believe `__all__` should have all exported values from this package, so would need to include the public functions defined in this file